go-sdk/sdk/centralized-configuration/centralized_configuration_delete_test.go

```diff
@@ -3,16 +3,15 @@
 package centralizedconfiguration_test
 
 import (
-	centralizedConfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
-	"github.com/konstellation-io/kai-sdk/go-sdk/sdk/messaging"
+	centralizedconfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```

Comment:
```suggestion
	centralizedConfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```

Use camelCase for Go.